packages/gatsby-source-npm-package-search/src/gatsby-node.js

```diff
@@ -28,8 +48,29 @@ exports.sourceNodes = async (
     hitsPerPage: 1000,
   })
 
-  hits.forEach(hit => {
+  await hits.map(async hit => {
```

Comment:
Should this be `await Promise.all(hits.map(/* foo bar baz */))`?

Reply:
Oh yeah my bad, I refactored it before pushing 🤦‍♂️